.github/workflows/links.yml

```
@@ -14,11 +14,19 @@ jobs:
     - name: Checkout Repo
       uses: actions/checkout@v4
 
+
+    - name: Restore lychee cache
+      uses: actions/cache@v4
+      with:
+        path: .lycheecache
+        key: cache-lychee-${{ github.sha }}
+        restore-keys: cache-lychee-
+
     - name: Link Checker
       id: lychee
       uses: lycheeverse/lychee-action@v1.9.3
       with:
-          args: --max-concurrency 5 .
+          args: --max-concurrency 5 --cache --max-cache-age 1d .
```

```suggestion
          args: --max-concurrency 1 --cache --max-cache-age 1d .
```

How about this to reduce the possibility of rate limiting? 

This job can run longer as it is a scheduled (or manually triggered) one.